adafruit_httpserver/response.py

```diff
@@ -73,8 +94,8 @@ def route_func(request):
     """
     Defaults to ``text/plain`` if not set.
 
-    Can be explicitly provided in the constructor, in `send()` or
-    implicitly determined from filename in `send_file()`.
+    Can be explicitly provided in the constructor, in ``send()`` or
+    implicitly determined from filename in ``send_file()``.
```

Comment:
The documentation builder _should_ be able to use single backticks and produce cross references to other methods/functions here. Did that fail?

Reply:
I had problems with single ticks as it often couldn't find the member in X in module Y

Reply:
Not a big deal; I've also found it doesn't work under strange circumstances.

Reply:
Are there any more changes that you would like me to include?